test/integration/pipeline/tx_pipeline_integration_test.cpp

```diff
@@ -17,6 +17,7 @@
 
 #include "integration/pipeline/tx_pipeline_integration_test_fixture.hpp"
 
+// TODO: refactor services to allow dynamic port binding IR-741
```

Comment:
There is need to refactor test or code?

Reply:
Firstly code then tests. I can add TODO to code as well, but I think this todo in tests will allow tracing problem better.